tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/routing/RedirectTest.java

```diff
@@ -45,37 +45,30 @@
 import org.apache.activemq.artemis.tests.extensions.parameterized.Parameters;
 import org.apache.activemq.artemis.core.server.routing.KeyType;
 import org.apache.activemq.artemis.core.server.cluster.impl.MessageLoadBalancingType;
-import org.apache.activemq.artemis.tests.util.Wait;
 import org.junit.jupiter.api.TestTemplate;
 import org.junit.jupiter.api.extension.ExtendWith;
 
 @ExtendWith(ParameterizedTestExtension.class)
 public class RedirectTest extends RoutingTestBase {
+   static final String[] protocols = new String[] {AMQP_PROTOCOL, CORE_PROTOCOL, OPENWIRE_PROTOCOL};
+   static final String[] pools = new String[] {CLUSTER_POOL, DISCOVERY_POOL, STATIC_POOL};
 
    @Parameters(name = "protocol: {0}, pool: {1}")
    public static Collection<Object[]> data() {
-      final String[] protocols = new String[] {AMQP_PROTOCOL, CORE_PROTOCOL, OPENWIRE_PROTOCOL};
-      final String[] pools = new String[] {CLUSTER_POOL, DISCOVERY_POOL, STATIC_POOL};
       Collection<Object[]> data = new ArrayList<>();
 
-      for (String protocol : Arrays.asList(protocols)) {
-         for (String pool : Arrays.asList(pools)) {
-            data.add(new Object[] {protocol, pool});
-         }
+      for (String pool : Arrays.asList(pools)) {
+         data.add(new Object[] {AMQP_PROTOCOL, pool});
```

Comment:
The protocol parameter isnt being used, the field it populated was removed, so seems like it can just go away rather than be passed a fixed unused value of AMQP_PROTOCOL, which will make for misleading test display names in results.

Reply:
hmmm... I made the same note.. didn't realize you had already done it.

Reply:
The comment timestamp is from when I originally made the comment, but I only actually completed and posted my review moments before you did yours, i.e we posted the reviews at essentially the same time (its only shown to the minute, so both have same timestamp value).

Reply:
I'll remove the protocol parameter.